dockerfile-image-update-cmdtool/src/main/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGithubUtil.java

```diff
@@ -194,20 +192,24 @@ public void updateStore(String store, String img, String tag) throws IOException
             String ownerOrg = myself.getLogin();
             storeRepo = githubUtil.getRepo(Paths.get(ownerOrg, store).toString());
         } catch (IOException e) {
-            storeRepo = githubUtil.createPrivateRepo(store);
+            storeRepo = githubUtil.createPublicRepo(store);
         }
-        updateStoreOnGithub(storeRepo, STORE_JSON_FILE, img, tag);
+        updateStoreOnGithub(storeRepo, Constants.STORE_JSON_FILE, img, tag);
     }
 
     protected void updateStoreOnGithub(GHRepository repo, String path, String img, String tag) throws IOException {
-        GHContent content;
         try {
-            content = repo.getFileContent(path);
+            repo.getFileContent(path);
```

Comment:
Is there a reason we need to to this once without the `latestCommit` and then do it again with `latestCommit`?

Reply:
The logic here essentially is "if file contents are missing, then create the store file in the catch block". I would love it if the lib allowed me to do this with an if statement, but unfortunately that is not the case.

Reply:
Also, I was hitting up a nasty behavior with the APIs, where if I didn't get the content with the latest commit, it would return the first commit. If there is no file yet, then most likely there is no commit in the repo and we'd get NPEs later.

Reply:
Seems like we might be able to use a `GHSearchBuilder` to do this in a less `catch`-y way. 😆  If you don't mind, it'd probably be nice to have a comment there stating the purpose and that another type of API call may be necessary.

Reply:
Something like https://github.com/username/image-tag-store-repo/search?utf8=%E2%9C%93&q=filename%3Astore.json+path%3A%2F&type= may do the trick